apps/meteor/server/settings/push.ts

```diff
@@ -107,6 +113,14 @@ export const createPushSettings = () =>
 				enableQuery: [],
 				secret: true,
 			});
+
+			await this.add('Push_google_api_credentials', '', {
+				type: 'code',
+				multiline: true,
+				enableQuery: [],
```

Comment:
Use `enableQuery` based on `Push_UseLegacy`'s value for both this new setting and the gcm one.